controllers/user.controller.js

```diff
@@ -156,7 +156,7 @@ exports.login = async (req, res) => {
         id: user._id,
```

Comment:
@akshitbansal108  are you talking about this?.

Reply:
Yes, like when we will send token to frontend on login/register, frontend can store user id along with token and send to the backend in headers whenever Anu api is hit.